models/articles-models.js

```diff
@@ -0,0 +1,18 @@
+const format = require("pg-format");
+const db = require("../db/connection.js");
+
+module.exports.fetchArticleByArticleId = (article_Id) => {
+  if (isNaN(article_Id)) {
+    return Promise.reject({ status: 400 });
+  } else {
```

Comment:
Because if the condition previous is met you return, you don't need an else here as when you return the function will terminate.